src/Spice86/Models/Debugging/CpuInstructionInfo.cs

```diff
@@ -4,8 +4,12 @@ namespace Spice86.Models.Debugging;
 
 using Iced.Intel;
 
+using Spice86.Core.Emulator.VM.Breakpoint;
+
 public partial class CpuInstructionInfo : ObservableObject {
     [ObservableProperty] private string? _stringRepresentation;
+    [ObservableProperty] private bool _hasBreakpoint;
+    [ObservableProperty] private BreakPoint? _breakPoint;
```

Comment:
Do we need both?
Can they get out of sync?

Reply:
We don't need both, I'll replace this entire pr with a dedicated BreakPoint management view and this ViewModel will only communicate with a IAddressBreakPointViewModel interface.